browser/net/brave_static_redirect_network_delegate_helper_unittest.cc

```diff
@@ -283,6 +283,27 @@ TEST_F(BraveStaticRedirectNetworkDelegateHelperTest, ModifySafeBrowsingURLV5) {
   EXPECT_EQ(ret, net::OK);
 }
 
+TEST_F(BraveStaticRedirectNetworkDelegateHelperTest,
+       ModifySafeBrowsingFileCheckURL) {
+  net::TestDelegate test_delegate;
+  GURL url(
+      "https://sb-ssl.google.com/safebrowsing/clientreport/download?"
+      "key=DUMMY_KEY");
+  std::unique_ptr<net::URLRequest> request = context()->CreateRequest(
+      url, net::IDLE, &test_delegate, TRAFFIC_ANNOTATION_FOR_TESTS);
+  std::shared_ptr<brave::BraveRequestInfo> before_url_context(
+      new brave::BraveRequestInfo());
+  brave::BraveRequestInfo::FillCTXFromRequest(request.get(),
+                                              before_url_context);
+  brave::ResponseCallback callback;
+  GURL::Replacements replacements;
+  replacements.SetHostStr(kBraveSafeBrowsingFileCheckProxy);
+  GURL expected_url(url.ReplaceComponents(replacements));
```

Comment:
minor: i think this test would be more precise if it tested for the literal expected string, like `https://sb-ssl.brave.com/...` instead of `url.ReplaceComponents(replacements)`

Reply:
done!